tests/providers/test_hetzner.py

```diff
@@ -30,13 +34,14 @@ def get_dns_cname_mock(self, request):
                             return_value=(_domain_mock, [], None)) as fixture:
                 yield fixture
 
-    @pytest.mark.skipif(no_network(), reason='No network, no dns.resolver call possible.')
-    @pytest.mark.ignore_get_dns_cname_mock('yes')
+    @pytest.mark.skipif(_no_dns_lookup(), reason='No DNS resolution possible.')
+    @pytest.mark.ignore_dns_cname_mock
     def test_get_dns_cname(self):
         """Ensure that zone for name can be resolved through dns.resolver call."""
-        _domain, _, _ = Provider._get_dns_cname('_acme-challenge.fqdn.{}.'.format(self.domain),
-                                                False)
+        _domain, _nameservers, _ = Provider._get_dns_cname(('_acme-challenge.fqdn.{}.'
```

Comment:
Can we assert something about the third element returned by the tuple?

Reply:
If present, the third element is the CNAME. Otherwise None (see docstring). Because I use productiv test domains, it could not be sure if there will be a CNAME sometime. So no, we can not assert something.

Reply:
Fair enough. Mock could be done to ensure a particular situation (record is a cname or not), but I do not see a real value to do it.

Reply:
Oh, sorry I did not see you found a way!